Mathlib/Topology/Algebra/LinearTopology.lean

```diff
@@ -0,0 +1,113 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir, María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández, Anatole Dedecker
+-/
+
+import Mathlib.RingTheory.TwoSidedIdeal.Operations
+import Mathlib.Topology.Algebra.Ring.Basic
+import Mathlib.Topology.Algebra.OpenSubgroup
+
+/-! # Linear topologies on rings
+
+Following Bourbaki, *Algebra II*, chapter 4, §2, n° 3, a topology on a ring `R` is *linear* if
+it is invariant by translation and admits a basis of neighborhoods of 0 consisting of
+two-sided ideals.
+
+- `IsLinearTopology.tendsto_mul_zero_of_left`: for `f, g : ι → R` such that `f i` converges to `0`,
+`f i * g i` converges to `0`.
+
+- `IsLinearTopology.tendsto_mul_zero_of_right`: for `f, g : ι → R` such that `g i` converges to `0`,
+`f i * g i` converges to `0`.
+
+-/
+
+open scoped Topology
+open Filter
+
+section Ring
+
+variable (α : Type*) [Ring α]
```

Comment:
I would prefer `R` over `alpha`.

Reply:
We can change that.

Reply:
Yes but it is not changed? There seem to be several conversations here which were resolved but no action was taken?

Reply:
I had made the push in the wrong branch…